spec/models/value_spec.rb

```diff
@@ -7,6 +7,52 @@
     it { is_expected.to be_a Hash }
   end
 
+  describe "##normalise_key (class method)" do
+    subject { described_class.normalise_key(key) }
+
+    context "when the key contains unsupported characters" do
+      let(:key) { "this-is!a{test}_key#123£567" }
+
+      it { is_expected.to be(:this_is_a_test__key_123_567) }
+
+      it { is_expected.to be_a Symbol }
```

Comment:
redundant given the first assertion?

Reply:
yes, technically so - I was trying to signify to a future dev that the keys must be symbols rather than strings. If its all the same, I think I'll leave this test in.